.github/workflows/publish.yml

```diff
@@ -0,0 +1,168 @@
+name: Publish ACA-Py with Plugins
+run-name: Publish ACA-Py ${{ github.event.release.tag_name || github.event.inputs.version }} with all plugins
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'ACA-Py version to build'
+        required: true
+        type: string
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build-and-publish:
+    strategy:
```

Comment:
For most of the other workflows in the openwallet-foundation repos we have a statement `if: github.repository == 'openwallet-foundation/acapy-plugins'` so these don't get run automatically when you have a fork.

Reply:
After discussion. This is fine to leave without a conditional.